android/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
   dependencies {
     classpath "com.android.tools.build:gradle:8.3.2"
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    if (kotlinVersion?.startsWith("2")) {
+    if (kotlin_version?.startsWith("2")) {
```

Comment:
did you validate this change?

I think the `kotlin_version` was not working so that is why it was left like this

Reply:
Yes, for me if I have `kotlinVersion ` it throws an error . That's why I changed it.
<img width="651" alt="Screenshot 2025-03-21 at 10 49 38" src="https://github.com/user-attachments/assets/375a7716-18d3-4ef4-aee2-bc3cfcef0c08" />

Reply:
Line 5 defines the kotlin version as `kotlin_version` - https://github.com/smileidentity/react-native/blob/7c0b287c1175936ad69727d3cd15b6fa81db6a76/android/build.gradle#L5